stereo_reconstruction:
  ros__parameters:

    hz: 1.0

    publishers:
      disparity_map: /disparity_map
      pointcloud: /pointcloud

    subscribers:
      raw_frame_left: /camera_left/raw_frame
      raw_frame_right: /camera_right/raw_frame

    enable_compute_pointcloud: True

    image_width: 1000
    image_height: 1000

    frames:
      stereo_link: camera_left_link